README.md

```diff
@@ -6,16 +6,8 @@ find a grammar they're interested in.
 
 ## Contributing
 
-If you have a grammar you're interested in contributing, please open an issue [here](https://github.com/tree-sitter-grammars/.github/issues).
+If you want to create a grammar, or have already made one that you'd like to contribute,
+please read the [guidelines] and open an issue [here][issues] when your grammar is ready.
```

Comment:
Do we want to add a link to the Matrix room? Or create a new, non-Neovim branded, "parser club" room? (For non-Discordians)